make lesson-check in testing

`make lesson-check` does not run on the CI check. This step is helpful for catching, e.g., missing blockquote types. This is a small but significant oversight.

Well, isn't that interesting. `make lesson-check-all`, which we run, *does not* throw an error when it catches things. `make lesson-check`, on the other hand, *does*.

```bash
$ make lesson-check-all
./README.md: Line(s) end with whitespace: 22, 32, 35
./index.md: Line(s) too long: 13
./index.md:18: Unknown or missing blockquote type None
./index.md:29: Unknown or missing blockquote type None
./index.md:31: Unknown or missing blockquote type None
./index.md:34: Unknown or missing blockquote type None
./setup.md: Line(s) too long: 114

$ make lesson-check
./index.md:18: Unknown or missing blockquote type None
./index.md:29: Unknown or missing blockquote type None
./index.md:31: Unknown or missing blockquote type None
./index.md:34: Unknown or missing blockquote type None
make: *** [Makefile:134: lesson-check] Error 1
```

--- a/.github/workflows/test_and_build.yml
+++ b/.github/workflows/test_and_build.yml
@@ -36,9 +36,12 @@ jobs:
         gem install bundler jekyll json kramdown
         bundle config set path '.vendor/bundle'
         bundle config build.nokogiri --use-system-libraries
-    - name: "Check lesson"
+    - name: "Check lesson for warnings"
       run: |
         make lesson-check-all
+    - name: "Check lesson for errors"
+      run: |
+        make lesson-check
     - name: "Check build"
       run: |
         make --always-make site

--- a/README.md
+++ b/README.md
@@ -19,7 +19,7 @@ writer, and then reviewed by the rest of the group once complete.
    [_includes/snippets_library](_includes/snippets_library). These snippets provide inputs and
    outputs that are host-specific and that are included automatically when the lesson website is
    built.
-    
+
    1. Code snippets are in files named `snippet_name.snip` and are included automatically
       when the lesson is built. For example, if the `snippet_name` was `login_output`,
       then the snippet file would be called `login_output.snip`.
@@ -29,10 +29,10 @@ writer, and then reviewed by the rest of the group once complete.
    3. In the episodes source, snippets are included using [Liquid](
       https://shopify.github.io/liquid/) scripting  `include` statements. For example, the first
       snippet in episode 12 is included using `{% include /snippets/12/info.snip %}`.
-      
+
 3. Edit `_config_settings.yml` in your snippets folder. These options set such things as the address
    of the host to login to, definitions of the command prompt, and scheduler names.
-   
+
 4. Add your snippet directory name to the GitHub Actions configuration file,
    [.github/workflows/test_and_build.yml](.github/workflows/test_and_build.yml).
 

--- a/setup.md
+++ b/setup.md
@@ -111,8 +111,9 @@ Git for Windows comes with SSH preinstalled: you do not have to do anything.
 
 > ## GUI Support
 >
-> If you know that the software you will be running on the cluster requires a graphical user interface (a GUI window needs to open for the application to run properly), please
-> install [MobaXterm](http://mobaxterm.mobatek.net) Home Edition.
+> If you know that the software you will be running on the cluster requires a graphical user
+> interface (a GUI window needs to open for the application to run properly), please install
+> [MobaXterm](http://mobaxterm.mobatek.net) Home Edition.
 {: .discussion}
 
 ### macOS

